opencog/atomspace/Atom.h

```diff
@@ -81,6 +81,7 @@ class Atom
     friend class TLB;             // Needs to view _uuid
     friend class CreateLink;      // Needs to call getAtomTable();
     friend class DeleteLink;      // Needs to call getAtomTable();
+    friend class ProtocolBufferSerializer; // Needs to de/ser-ialize an Atom
```

Comment:
If I remember correctly ._uuid, .getAtomTable() , probably some others. Part of implementation is still the same as it was before I came. For those accessors which are public now, I replace them with public ones, but some still are protected hence the 'friend request'. :)